templates/etc/permanent/notification-service.env

```diff
@@ -1,3 +1,4 @@
+NODE_ENV=production
```

Comment:
Specifically for notification-service: no, we expect devs to run that service on their host, not in their Vagrant instance.

For upload-service (which has the same change in a different file): yes, it would; I think that's good/fine, though, because the main behavior switch in the logging PRs is whether or not to include timestamps in log lines, and in both dev/staging/prod and in Vagrant, logging is routed through systemd which provides timestamps.